tests/dummy/public/test-forms/basic-fields/form.ttl

```diff
@@ -249,6 +249,17 @@ ext:conceptSchemeSelectorField a form:Field ;
 
 ext:mainFg form:hasField ext:conceptSchemeSelectorField .
 
+ext:conceptSchemeSelectorWithFormOptionPredicatesField a form:Field ;
+    sh:name "Select with form options" ;
+    sh:order 270 ;
+    sh:path ext:conceptSchemeSelectorValue ;
+    formOption:conceptScheme exampleConceptSchemes:foo-bar-baz ;
+    formOption:searchEnabled true ;
```

Comment:
Created prefix from the ext prefix

Reply:
Not sure what uri to take for the `formOptions` prefix

Ref: https://chat.semte.ch/group/form-builder?msg=Q7ETZb9rqAE569WHF